sys/ztimer64/ztimer64.c

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright (C) 2021 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2021 Freie Universität Berlin
+ *               2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     sys_ztimer
+ * @{
+ *
+ * @file
+ * @brief       ztimer64 core functionality
+ *
+ * This file contains ztimer64's main API implementation and functionality
+ *
+ * TODO:
+ * - add some kind of tick so base_clock overflows don't get lost
+ * - TEST
+ * - add proper unittests
```

Comment:
This will need to be squashed at some point, also I think we should document that there is currently no PM support in `ztimer64` in the same way `ztimer` does it.